targets/targets.json

```diff
@@ -3520,5 +3520,16 @@
         "device_name": "TMPM066FWUG",
         "detect_code": ["7011"],
         "release_versions": ["5"]
+    },
+    "WUNDERBAR": {
+        "inherits": ["MCU_K24F1M"],
+        "release_versions": ["5"],
+        "macros_add": ["TARGET_WUNDERBAR"],
+        "OUTPUT_EXT": "hex"
+    },
+    "WUNDERBAR_APP_BIN": {
```

Comment:
What is the intention with  this target. It defines output as binary, and adds bootloader?

Reply:
Yes - this target should be used if one doesn't want to overwrite existing bootloader on Wunderbar boards. Bin format is used to make it easier to upload to MCU (bootloader uses DFU and FW in bin format).

Reply:
My bad - I forgot that I have target override for target.mbed_app_start in my app's mbed_app.json. 

Is there any way to put it directly into target configuration (apart from ifdefs in linker)? The reason why I'm asking and why I think it would be best to have app offset set by target, is that Wunderbar's bootloader source is not available and the blob is available only when downloaded directly from board. I would like to spare users the pain of overwriting bootloader and need to solder debug header to unbrick Wunderbar (unless they know what they are doing of course).